modules/ROOT/pages/7.6.0-release-notes.adoc

```diff
@@ -107,12 +107,15 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 [[improvements]]
 == Improvements
 
-{productname} {release-version} also includes the following improvement<s>:
+{productname} {release-version} also includes the following improvements:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== Pressing Shift+Enter in Mentions in Comments dropdown should accept the active menu item
+// #TINY-11455
+
+In previous versions of {productname}, the `Shift+Enter` key was not properly handled when used with mentions in comments. This caused a new line to be inserted while the mentions dropdown remained open, leading to inconsistent behavior compared to mentions within the editor.
+
+In {productname} {release-version}, mentions in comments now handle the `Shift+Enter` key correctly by inserting the highlighted user into the comment text area and closing the dropdown. This ensures consistent functionality across the editor and comment areas.
```

Comment:
This needs to be moved under `Accompanying Premium plugin changes` for Comments plugin.

```
=== Comments

The {productname} {release-version} release includes an accompanying release of the **Comments** premium plugin.

**Comments** Premium plugin includes the following additions and improvement.

==== Title
// #TINY-xxxx

<Insert Here>

For information on the **Comments** plugin, see: xref:introduction-to-tiny-comments.adoc[Introduction to {companyname} Comments].

```

Reply:
@FarzadHayat Noted, so I will wait until your section is merged first, and then I will push my changes.